packages/discord.js/src/structures/GuildMember.js

```diff
@@ -425,8 +425,9 @@ class GuildMember extends Base {
 
   /**
    * Times this guild member out.
-   * @param {number|null} timeout The time in milliseconds
-   * for the member's communication to be disabled until. Provide `null` to remove the timeout.
+   * @param {number|null} timeout The duration in milliseconds
+   * for the member's communication to be disabled. The maximum duration allowed by discord is 28 days.
```

Comment:
I agree with the duration wording. However, we shouldn't document API limits in case they change (the version of the documentation may still reference incorrect limits).

Reply:
Thank you for the quick reply. Would it be ok to mention that a limit exists (or may exist) and link to discord's documentation?

Reply:
Since we don't do this anywhere else, I think it'd be best to leave for future discussion rather than now.